cirq-core/cirq/ops/gate_operation_test.py

```diff
@@ -27,7 +28,7 @@ def test_gate_operation_init():
 
 
 def test_invalid_gate_operation():
-    three_qubit_gate = cirq.ThreeQubitGate()
+    three_qubit_gate = ThreeQubitTestGate()
```

Comment:
Maybe we want to define `class ThreeQubitGate` here the way we do so in `cirq-google/cirq_google/optimizers/convert_to_sqrt_iswap_test.py` in this PR